README.md

```diff
@@ -82,7 +81,7 @@ and is compatible with C and C++ inputs.
 
 To manually compile a program in `foo.c` with Stabilizer, run:
 ```
-export SZ_CODE=1 SZ_HEAP=1 SZ_STACK=1 SZ_LOWER=1 SZ_LINK=1
+export SZ_CODE=1 SZ_HEAP=1 SZ_STACK=1 SZ_LINK=1
```

Comment:
Insignificant note: I believe that `SZ_LINK=1` is redundant with `SZ_CODE=1`, because it just reorders link order of object files.